src/lib/nodes/rate-info-service/RateInfoService.ts

```diff
@@ -0,0 +1,52 @@
+import { Node } from '@/lib/nodes/abstract.node'
+import axios, { AxiosInstance } from 'axios'
+import { HealthcheckResult } from '@/lib/nodes/types.ts'
+import { NODE_LABELS } from '@/lib/nodes/constants.ts'
+
+export type RateInfoResponse = {
+  success: boolean
+  date: number
+  result: Record<string, number>
+}
+
+export type RateHistoryInfoResponse = {
+  success: boolean
+  date: number
+  result: {
+    _id: string
+    date: number
+    tickers: Record<string, number>
+  }[]
+}
```

Comment:
These types belong to `RateInfoClient`. Move them to `/lib/nodes/rate-info-service/types/`